open-company/open-company-web
# Rich abstract

Review with:
- [Notify](https://github.com/open-company/open-company-notify/pull/14)

This make possible to tag people or to use rich format text in the abstract field. Also fixes the annoying problems with the post button tooltips (the tooltips are not always in sync with the content of the post).

To test:
- click New post
- [x] do you see a message about no headline when you hover the post button? Can't publish?
- now enter a title
- [x] no tooltip now on the post button? Good
- enter a long abstract (type until the tooltip is red)
- [x] do you see a message about the abstract too long when you hover the post button now? Can't post it?
- now mention someone in the abstract
- try all the formatters you want in the abstract
- mention the same user also in the body
- now publish the post
- [x] that user got only 1 notifications?
- go back to the previous user session
- [x] does the published message looks good in FoC?
- now enter quick post
- [x] do you see a message about no headline when you hover the post button? Can't publish the post?
- enter a headline
- enter fullscreen
- try to hack and publish the post with a long abstract or w/o a headline (play with the limits)
- publish another post with a rich format abstract
- [x] does it look good in FoC?
